frontend/TestRun/EventsTab.svelte

```diff
@@ -117,11 +117,29 @@
             return parsed;
         });
 
+        let sortedNemesises = nemesisData.sort((a, b) => a.start_time - b.start_time);
+        let nemesisIndex = 0;
+        for (let event of parsedEvents.sort((a, b) => a.time - b.time)) {
+            event.nemesis = undefined;
+            if ( sortedNemesises[nemesisIndex]) {
+                while(sortedNemesises[nemesisIndex]){
```

Comment:
Do you need outer if statement? if sortedNemesises[nemesisInded] is false whille loop also will not run